### ISSUE TITLE
Progress - updates needed based on updates in core

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1455]
The following issues were opened in core and should be addressed in the react component:

- Progress bar example update - https://github.com/patternfly/patternfly-next/issues/1128 
  - [x] Text strings in examples were updated.
  - [x] With this update, the react examples labeled "Progress (Labeled)" and "Progress (Dynamic Description)" should be merged into a single example, and should have aria attributes similar to the core examples "Progress Finite Step" and "Progress Step Instruction".
  - [x] It's important to note with these changes that the `aria-valuemax` should always be 100 unless a `label` prop is defined (which gets mapped to `aria-valuetext`). Using `aria-valuetext` is the only time we should allow the `aria-valuemax` attribute to be some value other than 100. See a11y documentation in the core example for more explanation about this.
- progress bar - move aria attributes - https://github.com/patternfly/patternfly-next/issues/1456
